### ISSUE TITLE
Some questions？

### ISSUE THREAD

**Issue by shenshenzhanzhan** [thomasjhuang/deep-learning-for-document-dewarping#7]
Thanks for this repo!
I have some questions about the results of different methods.
(1) Can pix2pixHD method gets a better result than DocUNet ? 
(2) Have you changed some things about the original pix2pixHD method?

**Comment by thomasjhuang** [comment]
1) I was using a custom dataset when trying to solve some problems and the results were not very successful. This was because there aren't many open source datasets to begin with, and it was challenging to make quality OCR data that was 'dewarped'. I found a problem in the dataset that I was working with regarding the 'format' of the text. DocUNet and works alike tended to use examples that were pages out of a standard novel or textbook where text was dense and neatly formatted. When applying synthetic augmentations to that data, I believe the work for the GAN was easier to 'unwarp' due to the relatively consistent 'format' so to speak compared to the data that I worked with which was paystubs, income statements and bank transactions. That data was particularly challenging to successfully dewarp on. I did find very good results in removing background noise with this model, where I could perform pretty well on the kaggle dataset I mention in the readme.

2) I haven't changed anything about the original pix2pixHD method, there is hyperparameter tuning I performed, and that in itself can impact the performance of your model significantly. However, hyperparameter tuning is a case by case basis, and so trying to apply my tune to your dataset might probably not work.
